Include/internal/pycore_ceval_state.h

```diff
@@ -78,13 +78,10 @@ struct _ceval_runtime_state {
 
 
 struct _ceval_state {
-    /* This single variable consolidates all requests to break out of
-     * the fast path in the eval loop.
-     * It is by far the hottest field in this struct and
-     * should be placed at the beginning. */
-    uintptr_t eval_breaker;
-    /* Avoid false sharing */
-    int64_t padding[7];
+    /* This variable holds the global instrumentation version. When a thread is
+       running, this value is overlaid onto PyThreadState.eval_breaker so that
+       changes in the instrumentation version will trigger the eval breaker. */
+    uintptr_t instrumentation_version;
```

Comment:
Now that this is just the instrumentation_version, did you want to make this a `uint32_t` and simplify `set_version_raw`?

Reply:
I decided to keep it as-is to keep the PR more focused. To keep things more consistent it would make sense to change the new `eval_breaker` to `uint32_t` as well, and that would make the change significantly bigger.